tensorly/regression/cp_plsr.py

```diff
@@ -151,17 +174,24 @@ def fit(self, X, Y):
                 self.Y_factors[1], T.index[:, component], comp_Y_factors_1
             )
 
+            B = T.lstsq(self.X_factors[0], T.reshape(comp_Y_factors_0, (-1, 1)))[0]
+            self.coef_ = T.index_update(
```

Comment:
As I'm reviewing this, given your other PR @aarmey I guess if we deprecate tensorflow we'll be able to remove those pesky index_updates (I think JAX supports the syntactic sugar now)